src/view/ChewingEditor.cpp

```diff
@@ -128,6 +130,57 @@ void ChewingEditor::exportUserphrase(const QString& file)
     emit model_->exportUserphrase(exporter);
 }
 
+void ChewingEditor::showAbout()
+{
+/* Make about dialog shows like aboutQt() on Mac OS X.
+   So we have to prevent call twice on Mac OS X. */
+#ifdef Q_OS_MAC
+    static QPointer<QMessageBox> oldAboutBox;
+
+    if (oldAboutBox) {
+        oldAboutBox->show();
+        oldAboutBox->raise();
+        oldAboutBox->activeWindow();
+        return;
+    }
+#endif
+
+    QString caption = QString("<h3>%1</h3><p>%2</p>")
+        .arg(tr("About Chewing Editor"))
+        .arg(tr("Version %1 (%2).").arg(PROJECT_VERSION).arg(PROJECT_GIT_VERSION));
+    QString text = QString("<p>%1</p><p>%2</p><p>%3</p><p>%4</p><p>%5</p><p>%6</p>")
```

Comment:
I tried your method but it shows like this (in first and second phrase), and it will be too closer with each paragraphs. Or should I use `\n\n` for this?

![image](https://cloud.githubusercontent.com/assets/841969/13996246/d3ed6e44-f167-11e5-82e7-8689f9ff1dbb.png)